sdk/recoveryservices-siterecovery/Azure.ResourceManager.RecoveryServicesSiteRecovery/api/Azure.ResourceManager.RecoveryServicesSiteRecovery.netstandard2.0.cs

```diff
@@ -5923,22 +5923,22 @@ public partial class VMwareCbtProtectionContainerMappingDetails : Azure.Resource
     {
         internal VMwareCbtProtectionContainerMappingDetails() { }
         public System.Collections.Generic.IReadOnlyList<string> ExcludedSkus { get { throw null; } }
-        public string KeyVaultId { get { throw null; } }
+        public Azure.Core.ResourceIdentifier KeyVaultId { get { throw null; } }
         public System.Uri KeyVaultUri { get { throw null; } }
         public System.Collections.Generic.IReadOnlyDictionary<string, int> RoleSizeToNicCountMap { get { throw null; } }
         public string ServiceBusConnectionStringSecretName { get { throw null; } }
-        public string StorageAccountId { get { throw null; } }
+        public Azure.Core.ResourceIdentifier StorageAccountId { get { throw null; } }
         public string StorageAccountSasSecretName { get { throw null; } }
         public string TargetLocation { get { throw null; } }
```

Comment:
Is this AzureLocation?

Reply:
Unable to confirm if it's `AzureLocation` or not, leave it as is.